### DISCUSSION TITLE
is currently Vercel down ??

### DISCUSSION THREAD

**Discussion by chauhanparth210** [vercel/vercel#5629] [Help]
I am not able to access Vercel.com from  my devices. I am form India. Thank you.

**Comment by soumyajit4419** [comment]
I am also facing the same issues. I am not able to access any website or API.

**Reply by chauhanparth210** [reply]
@soumyajit4419 hi, I am using VPN and it works fine now....

**Comment by sfsef** [comment]
I'm from Russia. It works nicely

**Comment by phips28** [comment]
down?

**Reply by sfsef** [reply]
No
